src/meta/user/userInvitationSummary.ts

```diff
@@ -1,3 +1,8 @@
-import { RoleName, UserRole } from 'meta/user/userRole'
+import { User } from 'meta/user/user'
+import { UserInvitation } from 'meta/user/userInvitation'
 
-export type UserInvitationSummary = UserRole<RoleName> & { email: string }
+export type UserInvitationSummary = UserInvitation & {
+  email: string
+  name: User['props']['name']
+  lang: User['props']['lang']
```

Comment:
Added two extra properties for UserInvitationSummary to simplify the display in CountryHome
We use `lang` when opening or copying the user invitation link